pkg/cloudprovider/cloudprovider_test.go

```diff
@@ -5,11 +5,12 @@ import (
 	"sort"
 	"testing"
 
-	"github.com/Azure/aad-pod-identity/pkg/config"
 	"github.com/Azure/go-autorest/autorest/azure"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"github.com/Azure/aad-pod-identity/pkg/config"
+
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-12-01/compute"
```

Comment:
This import should be grouped with the external import group.

Reply:
I don't know what you're asking, I ran `goimports` on the import path which is something that the linter complains about

Reply:
`github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-12-01/compute` should be above the `github.com/Azure/go-autorest/autorest/azure` import but `goimports` didn't catch it probably because it has a newline above it.

Reply:
Okay I'm going to get rid of the newlines and re-run `goimports`.